action.yaml

```
@@ -32,6 +32,9 @@ inputs:
   pr-number:
     description: 'Pull Request number on which to comment'
     required: false
+  post-comment:
+    description: 'Post comments on the pull request'
+    required: false
```

```suggestion
  post-comment:
    description: 'Post comments on the pull request'
    required: false
    default: true
```

This just makes the default obvious. I'm not sure if you can eliminate any of the default-value logic below.